platforms/nuget/TUTORIAL.md

```diff
@@ -0,0 +1,39 @@
+# Using the NuGet Package (OpenCVNuget) in your native C++ project
+This tutorial takes you through downloading the .nupkg file and setting it up in your C++ project, so you can begin writing C++ applications with OpenCV quickly.
+
+## Downloading the latest .nupkg (NuGet Package) file
+
+- You can download the package from
+    - The releases section on Github
+    (https://github.com/opencv/opencv/releases)
+    - The releases section on OpenCV website
+    (https://opencv.org/releases/)
+- Ensure that you have the correct package based on your Visual Studio or Microsoft Visual C++ Version. Currently supported versions are:
+    - Visual Studio 2019 - msvc 16.0
+    - Visual Studio 2018 - msvc 15.0
+    - Visual Studio 2017 - msvc 14.0
+
+## Setting up a local Package Source in the Visual Studio IDE
+
+- Make sure you have a *Source* directory, such as your Downloads folder setup, where you can keep all your .nupkg files
+- Start Visual Studio IDE
+- Create a new *Console App* Project
+- In the *Solution Manager*, right-click on References and select *Manage NuGet Packages*
+- A new NuGet window opens; click on the *settings icon* in the top right corner of the window
```

Comment:
Add Image guide here to simplify the description:

You could simplify steps 5 to 9 and just add a simpler description with the image:

- Add a local packages sources by as follows: 

![image](https://user-images.githubusercontent.com/24304201/130204437-9790941e-a3e9-437e-b1ee-ef0a967bece4.png)

- Click *Update*, then click on OK to close the pop-up window

Reply:
This has been changed for simplification.